### PR TITLE
introduce HttpClient with preconfigured timeouts and TTL

### PR DESCRIPTION
**Pull request by alexkorotkikh**
Issue #12

### CODE REVIEW

File: src/main/java/de/zalando/zmon/metriccache/HttpClientConfig.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+import java.util.concurrent.TimeUnit;
+
+@Configuration
+@ConfigurationProperties(prefix = "zmon.metricsconfig")
+public class HttpClientConfig {
+
+    private final int connectTimeout = 3000;
+    private final int socketTimeout = 30000;
+    private final int connTimeToLive = 120_000;
```
Comment (rajatparida86) [inline src/main/java/de/zalando/zmon/metriccache/HttpClientConfig.java:21]: is this intentional --> 120_000

Comment (alexkorotkikh) [reply src/main/java/de/zalando/zmon/metriccache/HttpClientConfig.java:21]: yes - it's to make it easier to read
> In Java SE 7 and later, any number of underscore characters (_) can appear anywhere between digits in a numerical literal. This feature enables you, for example, to separate groups of digits in numeric literals, which can improve the readability of your code.

© https://docs.oracle.com/javase/7/docs/technotes/guides/language/underscores-literals.html

**Comment by rajatparida86** [comment]
👍

**Comment by alexkorotkikh** [comment]
:+1:

**Comment by mohabusama** [comment]
👍

**Review by mohabusama — APPROVED** [review]

**Pull request merged**